changelog/21.0/21.0.0/summary.md

```diff
@@ -50,3 +50,13 @@ $ vtctldclient --server :15999 MoveTables --target-keyspace customer --workflow
 ```
 
 Mirror rules can be inspected with `GetMirrorRules`.
+
+### <a id="new-connection-drain"/>New Connection Drain
+
+We are adding a new way to terminate VTGate in v21 by using a connection drain timeout rather than an activity drain timeout.
+The goal of this new drain is to disallow new connections when VTGate is shutting down, but keep allowing idle/actives
+connections to execute queries until they disconnect or until the `--onterm_timeout` is reached.
```

Comment:
```suggestion
We added a new option to affect the VTGate shutdown process in v21 by using a connection drain timeout rather than the older activity drain timeout. The goal of this new behavior connection draining option is to disallow new connections when VTGate is shutting down, but continue allowing existing connections to finish their in flight work rather than killing them after the fixed `--onterm_timeout`.


This new behavior can be enabled by specifying the new `--mysql-server-drain-onterm` flag to VTGate.
```

Reply:
I did not follow the suggested text at 100%, but I improved it via 325c619fb3, here is what I wrote:

```
We added a new option to affect the VTGate shutdown process in v21 by using a connection drain timeout rather than the older activity drain timeout.
The goal of this new behavior, connection draining option, is to disallow new connections when VTGate is shutting down,
but continue allowing existing connections to finish their work until they manually disconnect or until the `--onterm_timeout` timeout is reached,
without getting a `Server shutdown in progress` error.

This new behavior can be enabled by specifying the new `--mysql-server-drain-onterm` flag to VTGate.
```